roles/fio_distributed/templates/server_vm.yml.j2

```diff
@@ -0,0 +1,63 @@
+---
+apiVersion: kubevirt.io/v1alpha3
+kind: VirtualMachineInstance
+metadata:
+  name: 'fio-server-{{item | string}}-{{ trunc_uuid }}'
+  namespace: '{{ operator_namespace }}'
+  labels:
+    app: fio-benchmark-{{ trunc_uuid }}
+spec:
+  domain:
+    devices:
+      disks:
+        - name: registrydisk
+        - name: cloudinitdisk
+{% if workload_args.storageclass is defined
+    or hostpath is defined %}
+        - name: data-volume
+          serial: data
+{% endif %}
+    resources:
+      requests:
+        memory: 5G
+  volumes:
+    - name: registrydisk
+      containerDisk:
+        image: {{ workload_args.vm_image | default('kubevirt/fedora-cloud-container-disk-demo:latest') }}
+    - name: cloudinitdisk
+      cloudInitNoCloud:
+        userData: |-
+          #cloud-config
+          password: fedora
+          chpasswd: { expire: False }
+          bootcmd:
+{% if workload_args.storageclass is defined
+    or hostpath is defined %}
+            - "mkdir -p {{ fio_path }} || true"
+            - mkfs.ext4 /dev/disk/by-id/ata-QEMU_HARDDISK_data
+            - "mount /dev/disk/by-id/ata-QEMU_HARDDISK_data {{ fio_path }}"
+{% endif %}
+          runcmd:
+            - dnf install -y fio wget curl python3 python3-pip
```

Comment:
Installing these packages in disconnected OCP env fails. Would be better if VM image had all necessary packages installed.

Reply:
Yes you're right... I've added a check if vm_image is defined... if so, it will not try to do these steps (and we assume the requirements are present)